### DISCUSSION TITLE
How do you create and set cookies to be logged in while trying to fetch paywalled full article content?

### DISCUSSION THREAD

**Discussion by CarlsonCarlson** [FreshRSS/FreshRSS#5500] [Q&A] [answered]
I want to put paywalled full article content in my FreshRSS (bloomberg).

I was looking at this https://www.reddit.com/r/rss/comments/12ppymj/newbie_question_can_an_rss_reader_parse_a/ and I’d like to set cookies that log me into bloomberg.

How do you create a cookie to be logged in while trying to fetch a bloomberg article? I see there's an option to “Use cookies when fetching the article content” with an example being “foo-bar; gdpr_consent=true; cookie=value” but I have no idea what any of that is.

Is that going to go in the URL when FreshRSS tries to fetch the article? What goes in that field?

**Comment by CarlsonCarlson** [comment]
Picture of the field I'm talking about
![pass cookies into freshRSS](https://github.com/FreshRSS/FreshRSS/assets/40775012/60f42fbc-8662-49ce-b4d7-dc437d18d4b7)

**Reply by Frenzie** [reply, marked as answer]
There are multiple ways to find out the value, but the easiest one is probably to open the developer tools, go to the network tab, and look at the headers.

![image](https://github.com/FreshRSS/FreshRSS/assets/202757/f33a3db3-f794-4479-a810-d0a722941d11)

**Comment by shahram7** [comment]
Does it work only when you have paid for the subscription, or can it bypass the paywall in general?

Is it possible to set the feeds in FreshRSS so that it always catches the full content? For cases where there is no paywall but the RSS feed only provides excerpts and you have to open the link for the full text, I am currently using the fivefilters-full-text-rss-docker app. I want to know if FreshRSS can do it too.

**Reply by Limezy** [reply]
Yes it can do it with the Readable extension https://github.com/printfuck/xExtension-Readable

**Comment by lprell** [comment]
Noob question. 

Does the cookies set on FreshRSS to retrieve full text on paywalled articles expire or it keep being refreshed somehow? If it does expire, is there any automated way to refresh or does it have to be manually changed after the previous one expired?

Thank you!
